src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -13,22 +13,34 @@ interface IVedtakProps {
 const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak }) => {
     const [brev, setBrev] = React.useState<string>('Genererer forhåndsvisning...');
     const [errorMessage, setErrorMessage] = React.useState<string | undefined>(undefined);
+    const [redirect, showRedirect] = React.useState<boolean>(false);
 
     React.useEffect(() => {
-        hentAktivVedtaksbrev(fagsak)
-            .then((response: Ressurs<string>) => {
-                if (response.status === RessursStatus.SUKSESS) {
-                    setBrev(response.data);
-                    setErrorMessage(undefined);
-                } else if (response.status === RessursStatus.FEILET) {
-                    setErrorMessage(response.melding);
-                } else {
-                    setErrorMessage('Ukjent feil, kunne ikke generere forhåndsvisning.');
-                }
-            })
-            .catch(error => {
-                setErrorMessage('Ukjent feil, Kunne ikke generere forhåndsvisning.');
-            });
+        if (
+            !!fagsak?.behandlinger
```

Comment:
Do you have a linter installed in your IDE? This is not valid typescript. 

I believe the easiest is to do something like I do in FastsettVedtak.tsx where i find the aktivVedtak. If this object is undefined we can display a redirect link, but the text should be something like `Vi finner ingen aktive vedtak på behandlingen, vennligst gå tilbake og fastsett vedtak.`.

And it could be a button not a link when checking this before doing the request I think.

Reply:
Yes I have a lint tool, and the code works well locally...

I agree with the back button and I saw you alreaday have added it. I will use your solution.